main/main.c

```diff
@@ -2339,7 +2339,7 @@ int php_module_startup(sapi_module_struct *sf, zend_module_entry *additional_mod
 		struct {
 			const long error_level;
 			const char *phrase;
-			const char *directives[17]; /* Remember to change this if the number of directives change */
+			const char *directives[19]; /* Remember to change this if the number of directives change */
```

Comment:
Maybe we can update this comment to be more understandable.

Reply:
I’m not sure how you want to make this comment more understandable, it’s very basic C

Reply:
I’m not sure how you want to make this comment more understandable, it’s very basic C

Reply:
I mean it was me doing a stupid mistake, no need to update this comment IMHO